multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -74,11 +78,14 @@ def default(presets: str = DEFAULT):
 
     presets, use_hpo = parse_presets_str(presets)
     if use_hpo:
+        default_tunable_hyperparameters, default_hyperparameter_tune_kwargs = get_default_hpo_setup()
         hyperparameters.update(default_tunable_hyperparameters)
         hyperparameter_tune_kwargs.update(default_hyperparameter_tune_kwargs)
 
     if presets in [HIGH_QUALITY, DEFAULT]:
         if use_hpo:
+            from ray import tune
```

Comment:
consider using `try_import_ray` for a more informative import error message.

```
from autogluon.core.utils.try_import import try_import_ray  # top of file


def import_ray_tune():
    try_import_ray()
    from ray import tune
    return tune


def my_func():
    tune = import_ray_tune()
    ...

```

Reply:
If using `try_import_ray ()`, the following error message shows.

`ImportError: ray is required to train folds in parallel. A quick tip is to install via `pip install ray==2.2.0`, or use sequential fold fitting by passing `sequential_local` to `ag_args_ensemble` when calling tabular.fitFor example: `predictor.fit(..., ag_args_ensemble={'fold_fitting_strategy': 'sequential_local'})``

which is more customized for tabular predictor and may bring confusions for AutoMM users.

Reply:
good point, then it is ok as is